_data/locales/tr-TR.yml

```
@@ -70,8 +70,21 @@ post:
     share_link:
       title: Linki kopyala
       succeed: Link kopyalandı.
+# Date time format.
+# See: <http://strftime.net/>, <https://day.js.org/docs/en/display/format>
+df:
+  post:
+    strftime: "%b %e, %Y"
+    dayjs: "ll"
+  archives:
+    strftime: "%b"
+    dayjs: "MMM"
 
 # categories page
 categories:
-  category_measure: kategori
-  post_measure: gönderi
+  category_measure:
+    singular: kategori
+    plural: kategori
+  post_measure:
+    singular: gönderi
+    plural: gönderi
```

When the singular and plural are the same, there is no need to repeat the definition, restore them.

```suggestion
  category_measure: kategori
  post_measure: gönderi
```